src/api/todos.ts

```diff
@@ -0,0 +1,18 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${6725}`);
+};
+
+export const postTodo = (data: unknown) => {
```

Comment:
please add proper typing for data
```suggestion
export const postTodo = (data: Omit<Todo, 'id'>) => {
```

Reply:
ok, thanks